crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -299,6 +348,11 @@ type MoveModuleEdge {
 	cursor: String!
 }
 
+type MoveModuleId {
+	name: String!
+	package: MovePackage
```

Comment:
The schema has this mandatory, but I think this makes more sense?

Reply:
i'll just follow the schema